src/Integration/Connection/MinimumSupportedSonarQubePlugin.cs

```diff
@@ -35,7 +35,10 @@ internal class MinimumSupportedSonarQubePlugin
         // i.e. the analysis rules implementations should be similar so the issues shown in the IDE should be
         // similar to those reported on the server.
         public static readonly MinimumSupportedSonarQubePlugin CFamily = new MinimumSupportedSonarQubePlugin(SonarPluginKeys.SonarCFamily, "SonarCFamily", "6.0", Language.Cpp, Language.C);
-        public static readonly IEnumerable<MinimumSupportedSonarQubePlugin> All = new[] { CSharp, VbNet, CFamily };
+
+        public static readonly MinimumSupportedSonarQubePlugin JsTs = new MinimumSupportedSonarQubePlugin(SonarPluginKeys.SonarJs, "SonarJS", "8.6", Language.Js, Language.Ts);
```

Comment:
I put version 8.6 as "minimal" because that's what we embed in SLVS in this release.
Perhaps it would be better to pick an older version, in cause the user wants to connect to a SQ version that has an older version. SC is at 9.4 btw.